### PR TITLE
ClusterLoader - Add unstructured support for runtimeobjects.go

### PR DESCRIPTION
**Pull request by Betula-L**
**Why we need it**:
https://github.com/kubernetes/perf-tests/issues/414

**What this PR does**:
Related to https://github.com/kubernetes/perf-tests/pull/423

### CODE REVIEW

**Comment by wojtek-t** [comment]
@krzysied - can you do a first pass of review here? I would like to also take a look later.

File: clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go [hunk 1]
```diff
@@ -160,12 +174,45 @@ func GetSelectorFromRuntimeObject(obj runtime.Object) (labels.Selector, error) {
 	}
 }
 
+// Note: This function assumes each controller has field Spec.Selector.
+// Moreover, Spec.Selector should be *metav1.LabelSelector, expect RelicationController.
```
Comment (krzysied) [inline clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go]: s/expect/except for

Comment (Betula-L) [reply clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go]: Done.

**Comment by Betula-L** [comment]
@wojtek-t Is that good to you?

**Comment by wojtek-t** [comment]
Yes - thanks!

/lgtm
/approve
/ok-to-test

**Pull request merged**